readme.md

```diff
@@ -136,7 +136,8 @@ Just do not define a read function.
 const multiplyCountAtom = atom(null, (get, set, by) => set(countAtom, get(countAtom) * by))
 
 function Controls() {
-  const [, multiply] = useAtom(multiplyCountAtom)
+  // causes unnecessary re-render: const [, multiply] = useAtom(multiplyCountAtom)
+  const nultiply = useSetAtom(multiplyCountAtom)
```

Comment:
Please note that `useSetAtom` from `jotai` is nothing new. We have `useUpdateAtom` from `jotai/utils` from the beginning with the same functionality and with the same mind.